Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.py

```diff
@@ -611,56 +908,70 @@ def main() -> None:
     :return:
     :rtype:
     """
+    # get pack version
+    response = demisto.internalHttpRequest("GET", "/contentpacks/metadata/installed")
+    packs = json.loads(response["body"])
 
-    api_key = demisto.params().get('apikey')
-    proxy = demisto.params().get('proxy', False)
+    pack_version = "0.0.0"
+    for pack in packs:
+        if pack["name"] == "GreyNoise":
+            pack_version = pack["currentVersion"]
 
-    demisto.debug(f'Command being called is {demisto.command()}')
+    api_key = demisto.params().get("apikey")
+    proxy = demisto.params().get("proxy", False)
+
+    demisto.debug(f"Command being called is {demisto.command()}")
     try:
 
         client = Client(
             api_key=api_key,
             api_server=API_SERVER,
             timeout=TIMEOUT,
-            proxy=handle_proxy('proxy', proxy).get('https', ''),
+            proxy=handle_proxy("proxy", proxy).get("https", ""),
             use_cache=False,
-            integration_name=f"xsoar-integration-v{demisto.demistoVersion()['version']}"
+            integration_name=f"xsoar-integration-v{pack_version}",
         )
 
-        if demisto.command() == 'test-module':
+        if demisto.command() == "test-module":
             # This is the call made when pressing the integration Test button.
             result: Any = test_module(client)
             return_results(result)
 
-        elif demisto.command() == 'greynoise-ip-quick-check':
+        elif demisto.command() == "greynoise-ip-quick-check":
             result = ip_quick_check_command(client, demisto.args())
             return_results(result)
 
-        elif demisto.command() == 'ip':
+        elif demisto.command() == "ip":
             result = ip_reputation_command(client, demisto.args())
             return_results(result)
 
-        elif demisto.command() == 'greynoise-stats':
+        elif demisto.command() == "greynoise-stats":
             result = stats_command(client, demisto.args())
             return_results(result)
 
-        elif demisto.command() == 'greynoise-query':
+        elif demisto.command() == "greynoise-query":
             result = query_command(client, demisto.args())
             return_results(result)
 
-        elif demisto.command() == 'greynoise-riot':
+        elif demisto.command() == "greynoise-riot":
             result = riot_command(client, demisto.args())
             return_results(result)
+
+        elif demisto.command() == "greynoise-context":
+            result = context_command(client, demisto.args())
+            return_results(result)
     # Log exceptions and return errors
-    except DemistoException as err:
-        return_error(str(err))
+    except RequestFailure:
+        raise DemistoException(EXCEPTION_MESSAGES["UNAUTHENTICATED"])
```

Comment:
Why did you remove the `except DemistoException as err` line?
It is a best practice

Reply:
This was the linter throwing error and was the only way I could figure out how to resolve this.  I'm happy to roll this back if that is what is desired.